src/rgw/rgw_swift_auth.cc

```diff
@@ -203,13 +201,17 @@ class TempURLEngine::PrefixableSignatureHelper
   const boost::optional<const std::string&> prefix;
 
 public:
-  PrefixableSignatureHelper(const std::string& decoded_uri,
+  PrefixableSignatureHelper(const std::string& _decoded_uri,
```

Comment:
@rzarzynski Is there an convention to prefix underscore in string name. 
Though I am not aware. Or need for rados gateway. Sorry for that